css/component/utility.css

```diff
@@ -28,21 +28,7 @@
 .vertical-direction{
     flex-direction: column;
 }
-.btn{
-    cursor: pointer;
-}
```

Comment:
You can instead reset for this instead of making a class

Reply:
ok! I will do that.